sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,172 @@
+- Added overloads with `sycl::queue` for some functions in [`sycl_ext_oneapi_bindless_images`](https://github.com/intel/llvm/blob/4d0c552985bd0e4fc16740a0fc89393ef7fe3d04/sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc) [4d0c552985bd]
+- Specify an error in `sycl_ext_oneapi_graph` if an executable graph is submitted to a queue with a different device or context [2c53815c6c1e]
+
+## Bug Fixes
+### SYCL Compiler
+- Fixed backend option passing issues with multiple targets [12ca2db0e30b]
+- Fixed `-fgpu-rdc` option for CUDA [f7595ac7527e]
+- Fixed triple in offload mismatch warning [0fd9a4eb17a2]
+- Fixed a compilation issue when `-enable-global-offset=false` for AMDGU and NVPTX targets [00cf4c29740b]
+
+### SYCL Library
+- Fixed stack corruption in `sycl::modf` for `sycl::half` type [7c6541dc18e3]
+- Fixed `nextafter(half)` precision on host [1f10f3588e8c]
+- Fixed identity values for (sub)group collectives min/max for AMD and CUDA [248389371932]
+- Corrected exception thrown when kernel is submitted with a large (unsupported) `reqd_work_group_size` attribute [25c3666dffa9]
+- Implemented hip `prefetch`/`mem_advise` (noop) for rocm < 5.0 [7e4cc1113987]
+- Adjusted `range` definition so it has a default constructor in accordance with the SYCL spec [0289d2ac94f9]
+- Added missing return statement in the `multi_ptr<void..>` conversion operator [836469ba88e2]
+- Optimized kernel fusion implementation so that some resources can be released earlier [a33b0a950ae6]
+- Fixed an integer overflow that happened when sub-buffer size exceeds `INT_MAX` [2fb0b140ad3b]
+- Fixed `std::enable_if` typo for `marray` and `multi_ptr` [0ef25ec3bdf8]
+- Fixed an issue with `reqd_work_group_size` attribute values bigger than `INT_MAX` being incorrectly handled [cee07d3d2652]
+- Fixed UB in group sort scratch memory access [ed4c01301a7c]
+- Use primary context in HIP adapter [d1c92cb95b29]
+- Fixed a crash when default-constructed accessor is passed to a kernel [fb8ddbf7b0ea]
+- Stopped `sycl_ext_oneapi_graph` subgraph nodes from creating incorrect dependencies on subsequent usages of the subgraph [92ddf8d562d7]
+- Fixed ambiguous `log2` call in host code [12fa0af28e88]
+
+### Documentation
+- Updated `device_image_scope` property  in `sycl_ext_oneapi_device_global` [be8e031cd078]
+
+## API/ABI breakages
+- Deprecated experimental functions for ESIMD: `quot`, `div`, `mod`, `dp*`, `line`, `lrp` [76976a22ba2e]
+- Removed ESIMD Emulator plugin [1ece6da6b113]
+- Deprecated `sycl::abs(genfloat)` [d15d4409b2cf]
+
+## Upcoming API/ABI breakages
+- Fixed sycl::vec ABI inconsistencies and made it trivially copyable if `-fpreview-breaking-changes` is specified  [531aabfdee91] [f4ed132f243a] 
+- Updated `sycl.hpp` to not include `<complex>` and `<cmath>` if `-fpreview-breaking-changes` is set. [6ed0ab887d18] [d22c9c58d2e7]
```

Comment:
I think we could have a standalone paragraph under this section saying that those changes are only in effect if `-fpreview-breaking-changes` flag is passed. This would allow to simplify wording of each item in the section.